e2etests/testdata/stable/people/dagre/board.exp.json

```diff
@@ -294,10 +294,10 @@
       "type": "person",
       "pos": {
         "x": 0,
-        "y": 280
+        "y": 260
       },
-      "width": 18,
-      "height": 21,
+      "width": 41,
+      "height": 61,
```

Comment:
the padding isn't set for either dimensions when a dimension is set because even the opposite dimension can affect the minimum size of a shape. See this person shape which had a desiredWidth of 16 but now is much further from that due to the padding on the y-axis.